src/MOI_wrapper.jl

```diff
@@ -117,8 +117,8 @@ end
 
 function MOI.supports_constraint(
     ::Optimizer{T},
-    ::Type{MOI.ScalarNonlinearFunction},
-    ::Type{<:Union{MOI.EqualTo{T},MOI.GreaterThan{T},MOI.LessThan{T}}},
+    ::Type{MOI.VectorNonlinearFunction},
+    ::Type{<:MOI.AbstractVectorSet},
```

Comment:
We should probably have a list of sets for which we define `vexity` instead because otherwise, if there is a set for which `vexity` is not defined but there is a bridge to convert it into sets for which `vexity` is defined, it won't be applied because Convex lied saying that `vexity` is defined

Reply:
I added a comment

Reply:
We'll say `true`, and then end up calling a missing vexity and get ```"`Convex.vexity(vex, ::$(typeof(set)))`: is not yet implemented. Please open an issue at https://github.com/jump-dev/Convex.jl"```, so that seems like an okay error. It tells them to open an issue, at least.